packages/xo-web/src/common/intl/messages.js

```diff
@@ -980,6 +980,7 @@ const messages = {
   // ----- host stat tab -----
   statLoad: 'Load average',
   // ----- host advanced tab -----
+  disksSystemHealthy: 'All disks are healthy ✔️',
```

Comment:
I would prefer ✅ instead. Otherwise, an icon in the UI directly after the translated text is also fine (still something green like 🟢)